ql/src/java/org/apache/hadoop/hive/ql/exec/tez/YarnQueueHelper.java

```diff
@@ -51,41 +51,41 @@ public class YarnQueueHelper {
   private int lastKnownGoodUrl;
   private boolean sslForYarn;
   private boolean isHA;
-  private static String webapp_conf_key = YarnConfiguration.RM_WEBAPP_ADDRESS;
-  private static String webapp_ssl_conf_key = YarnConfiguration.RM_WEBAPP_HTTPS_ADDRESS;
-  private static String yarn_HA_enabled = YarnConfiguration.RM_HA_ENABLED;
-  private static String yarn_HA_rmids = YarnConfiguration.RM_HA_IDS;
+  private static String WEBAPP_CONF_KEY = YarnConfiguration.RM_WEBAPP_ADDRESS;
+  private static String WEBAPP_SSL_CONF_KEY = YarnConfiguration.RM_WEBAPP_HTTPS_ADDRESS;
+  private static String YARN_HA_ENABLED = YarnConfiguration.RM_HA_ENABLED;
+  private static String YARN_HA_RMIDS = YarnConfiguration.RM_HA_IDS;
```

Comment:
these can be final

Reply:
directly used which are defined in YarnConfiguration.